data/aryasoni98.json

```
@@ -0,0 +1,34 @@
+{
+    "name": "Arya Soni",
+    "displayStatsPublic": true,
+    "type": "personal",
+    "bio": "Building Startup | NLP Intern @IITP'22 | NSUT'21 Intern | DevOps | ML Enthusiast",
+    "avatar": "https://github.com/aryason98.png",
```

Please fix your avatar link with the correct username!

```suggestion
    "avatar": "https://github.com/aryasoni98.png",
```